drud/ddev
# Shopware6 support for ddev, fixes #1988

## The Problem/Issue/Bug:

#1988 requests Shopware support for ddev. This PR provides support for Shopware 6

## How this PR Solves The Problem:

Adds support for Shopware 6

## Manual Testing Instructions:

Follow the [quickstart in the docs](https://github.com/drud/ddev/pull/2551/files#diff-6e74013bd43a95752e547c952a19faff266f253e752f719d513e74f4bbe1e74aR351-R376)

You should be able to get a basic install going.

## Automated Testing Overview:

A shopware6 stanza was added to the testable sites.
